### PR TITLE
Only normalize #72777a fills

### PR DESCRIPTION
**Pull request by ryantmer**
If an icon has fills of other colours (like the checkboxes), allow them - otherwise the entire icon is one single-colour blob.

### CODE REVIEW

File: tier2-icons.html [hunk 1]
```diff
@@ -108,18 +108,18 @@
           <path fill-rule="evenodd" d="M24 4v2a1 1 0 0 1-2 0V4a2.006 2.006 0 0 0-2-2h-2a1 1 0 0 1 0-2h2a4 4 0 0 1 4 4zm0 14v2a4 4 0 0 1-4 4h-2a1 1 0 0 1 0-2h2a2.006 2.006 0 0 0 2-2v-2a1 1 0 0 1 2 0zM7 1a1 1 0 0 1-1 1H4a2.006 2.006 0 0 0-2 2v2a1 1 0 0 1-2 0V4a4 4 0 0 1 4-4h2a1 1 0 0 1 1 1zm0 22a1 1 0 0 1-1 1H4a4 4 0 0 1-4-4v-2a1 1 0 0 1 2 0v2a2.006 2.006 0 0 0 2 2h2a1 1 0 0 1 1 1zm3-15.5l6 4.8-6 4.2v-9m0-2a2 2 0 0 0-2 2v9a2 2 0 0 0 3.147 1.638l6-4.2a2 2 0 0 0 .1-3.2l-6-4.8A2 2 0 0 0 10 5.5z"/>
         </g>
         <g id="change-file">
-          <g fill-rule="evenodd">
+          <g fill="#73787B" fill-rule="evenodd">
```
Comment (ryantmer) [inline tier2-icons.html]: I strongly suspect this, `navigate`, and `self-assessment` are supposed to be #72777A, but I will verify with Sarah.

Comment (ryantmer) [inline tier2-icons.html]: Confirmed that they are supposed to be #72777A, so I'll update them.

File: cli/iconset-builder.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ var iconsetObjTemplate = {
 };
 
 function processFill(elem) {
-	if (elem.$ && elem.$.fill && elem.$.fill !== 'none') {
+	if (elem.$ && elem.$.fill && elem.$.fill !== 'none' && elem.$.fill.toLowerCase() === '#72777a') {
```
Comment (dlockhart) [inline cli/iconset-builder.js]: You can likely remove the check for `none` now that it checking specifically for that color.

File: images/tier2/change-file.svg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
```
Comment (dlockhart) [inline images/tier2/change-file.svg]: Did you intend to modify these 3 SVGs at the same time? Seems like a separate thing maybe for a separate PR?

File: tier2-icons.html [hunk 4]
```diff
@@ -114,12 +114,12 @@
           </g>
         </g>
         <g id="check-box-unchecked">
```
Comment (dlockhart) [inline tier2-icons.html]: Nice! These were the expected results, so good stuff. 👍

**Comment by ryantmer** [comment]
Tried this out - as expected, the checkmark in the selected checkbox changes colour when a new colour is applied, but the box and background remain their associated colours of grey.

File: cli/iconset-builder.js (hunk 2 above)
Comment (ryantmer) [inline cli/iconset-builder.js]: Ah, yeah truth. Will do.

File: images/tier2/change-file.svg (hunk 3 above)
Comment (ryantmer) [inline images/tier2/change-file.svg]: Yeah fair - I checked with Sarah and they are indeed supposed to be #72777a, but I can do that in a separate one.

File: tier2-icons.html (hunk 1 above)
Comment (ryantmer) [inline tier2-icons.html]: These are set to an incorrect grey, hence them now showing up here - I've fixed them in #101.

**Comment by dlockhart** [comment]
LGTM, thanks!
